webclient/app/src/app/commons/entityCard/Entity.js

```diff
@@ -0,0 +1,36 @@
+interface EntityDataType {
```

Comment:
I think this structure would be better handled in a separate folder than in entityCard. Maybe a structures folder would be more suitable

Reply:
the structure doesn't suit at all for me. We can discuss this later.